### PR TITLE
Update link to ASP.NET porting guide ebook to the HTML version

### PR DESCRIPTION
**Pull request by danroth27**
I find it a bit jarring to have to download a file to read content on the web. Is there any particular reason we prefer to link to the PDF? If not, then I think we should link to the HTML version.

@nishanil

### CODE REVIEW

**Comment by guardrex** [comment]
@danroth27 ... There was an update to this text on another PR ... that's why the merge conflict 💥 hit. https://github.com/dotnet/AspNetCore.Docs/pull/25992/files

@Rick-Anderson ... btw ... one thing that I just noticed on that PR ... there's an `en-us` in one of the links that can be removed.

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson ... btw ... one thing that I just noticed on that PR ... there's an en-us in one of the links that can be removed.

That's fine. You get a warning with `/en-us` that tells you how to suppress the warning by appending a query string when it's needed.

File: aspnetcore/migration/proper-to-2x/index.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ uid: migration/proper-to-2x/index
 
 By [Isaac Levin](https://isaaclevin.com)
 
+=======
 This article serves as a reference guide for migrating ASP.NET apps to ASP.NET Core.
 
 The [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant) is a is a command-line tool that can help migrate ASP.NET to ASP.NET Core. For more information, see  [Overview of the .NET Upgrade Assistant](/dotnet/core/porting/upgrade-assistant-overview) and [Upgrade an ASP.NET MVC app to .NET 6 with the .NET Upgrade Assistant](/dotnet/core/porting/upgrade-assistant-aspnetmvc).
```
Comment (Rick-Anderson) [inline aspnetcore/migration/proper-to-2x/index.md:17]:
```suggestion
The [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant) is a is a command-line tool that can help migrate ASP.NET to ASP.NET Core. For more information, see  [Overview of the .NET Upgrade Assistant](/dotnet/architecture/porting-existing-aspnet-apps/) and [Upgrade an ASP.NET MVC app to .NET 6 with the .NET Upgrade Assistant](/dotnet/core/porting/upgrade-assistant-aspnetmvc).
```

File: aspnetcore/migration/proper-to-2x/index.md [hunk 2]
```diff
@@ -11,9 +11,10 @@ uid: migration/proper-to-2x/index
 
 By [Isaac Levin](https://isaaclevin.com)
 
+=======
```
Comment (Rick-Anderson) [inline aspnetcore/migration/proper-to-2x/index.md:14]:
```suggestion
=======
```

Comment (Rick-Anderson) [inline aspnetcore/migration/proper-to-2x/index.md:14]:
```suggestion
```

**Comment by Rick-Anderson** [comment]
@danroth27  I fixed the merge conflict. Is this what you wanted?

**Comment by danroth27** [comment]
@Rick-Anderson Yup, LGTM.

**Pull request merged**